src/database/DataTypes/StocktakeItem.js

```diff
@@ -142,8 +142,8 @@ export class StocktakeItem extends Realm.Object {
    * Finds the mode option within this stocktake items batches
```

Comment:
could use an update

Reply:
Interestingly, the return definition is already correct! Was the intent to return the string originally?

Reply:
lets go with that!

Reply:
https://github.com/openmsupply/mobile/pull/1008/commits/d04a3ebf1bdee015e1e35b1aabf0aad43cd01f23